src/editor/app.js

```diff
@@ -390,6 +390,26 @@ UiDriver.registerEventHandler("C_CMD_SET_THEME", function(msg, data, prevReturn)
     editor.setOption("theme", data.name);
 });
 
+UiDriver.registerEventHandler("C_CMD_SET_FONT", function(msg, data, prevReturn) {
+
+    var fontSize = (data.size != "" && data.size>0) ? ("font-size:" + data.size + "px;") : "";
+
+    var fontFamily = data.family != "" ? ("font-family:'" + data.family + "';") : "";
+
+    var styleTag = document.getElementById('userFont');
+
+    if( styleTag  ){ 
+        styleTag.innerHTML = ".editor > .CodeMirror { " + fontFamily + fontSize + " }";
+        styleTag.appendTo('head');
```

Comment:
Actually, it would probably be best to change the styles directly like I 
did for the symbols CodeMirror add-on.  This removes the need to keep 
appending tags to override the previous one.